.travis.yml

```diff
@@ -14,7 +14,7 @@ services:
 before_script:
```

Comment:
I think we might be able to just remove all these at some point. The duplication is probably not worth it.

Reply:
Yes this is the same discussion than we had for comit-rs.

Reply:
Sorry, did not see this in time but created an issue for it.